benchmark/assert/deepequal-set.js

```diff
@@ -33,7 +33,7 @@ function benchmark(method, n, values, values2) {
 function main({ n, len, method, strict }) {
   const array = Array(len).fill(1);
 
-  var values, values2;
+  let values, values2;
```

Comment:
nit: these variables can be moved as const inside each case (since they're strictly used within the scope of each case)